src/R/Core/Impl/Microsoft.R.Core.csproj

```diff
@@ -79,6 +80,12 @@
     <Compile Include="AST\Utility\AstWriter.cs" />
     <Compile Include="AST\Values\MissingValue.cs" />
     <Compile Include="AST\Values\NullValue.cs" />
+    <Compile Include="Install\IMicrosoftRClientInstaller.cs" />
+    <Compile Include="Install\ISupportedRVersionRange.cs" />
```

Comment:
Perhaps create new assembly specific to R engine itself? Microsoft.R.Engine? Or move to Support and I will take RData things out of it and into the editor.